features/fixtures/mazerunner/jvm-scenarios/src/main/java/com/bugsnag/android/mazerunner/scenarios/JvmAnrSleepScenario.kt

```diff
@@ -21,12 +20,6 @@ internal class JvmAnrSleepScenario(
 
     override fun startScenario() {
         super.startScenario()
-        val main = Handler(Looper.getMainLooper())
-        main.postDelayed(
-            Runnable {
-                Thread.sleep(100000)
```

Comment:
This scenario is designed specifically to test how we deal with stack traces of syscall-generating Java methods like `Thread.sleep()`. Changing it to something else will render it useless, unfortunately.

Reply:
Ah, ok - good spot.  I'll change it back and add a comment to that effect so I don't swoop in another day and try to put it back!  If it's the only such scenario it's far from the end of the world and I'll try to put some smarter logic in for the `skip`.